wpimath/src/test/native/cpp/estimator/SwerveDrivePoseEstimatorTest.cpp

```diff
@@ -215,3 +216,57 @@ TEST(SwerveDrivePoseEstimatorTest, BadInitialPose) {
     }
   }
 }
+
+TEST(SwerveDrivePoseEstimatorTest, SimultaneousVisionMeasurements) {
```

Comment:
There should be documentation stating what these tests are intended to check and what the intended behavior is.